ReactiveCocoa/Objective-C/NSURLConnection+RACSupport.m

```diff
@@ -16,39 +16,37 @@ @implementation NSURLConnection (RACSupport)
 
 + (RACSignal *)rac_sendAsynchronousRequest:(NSURLRequest *)request {
 	NSCParameterAssert(request != nil);
-
+	
 	return [[RACSignal
-		createSignal:^(id<RACSubscriber> subscriber) {
-			NSOperationQueue *queue = [[NSOperationQueue alloc] init];
-			queue.name = @"org.reactivecocoa.ReactiveCocoa.NSURLConnectionRACSupport";
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-			[NSURLConnection sendAsynchronousRequest:request queue:queue completionHandler:^(NSURLResponse *response, NSData *data, NSError *error) {
-				// The docs say that `nil` data means an error occurred, but
-				// `nil` responses can also occur in practice (circumstances
-				// unknown). Consider either to be an error.
-				//
-				// Note that _empty_ data is not necessarily erroneous, as there
-				// may be headers but no HTTP body.
-				if (response == nil || data == nil) {
-					[subscriber sendError:error];
-				} else {
-					[subscriber sendNext:RACTuplePack(response, data)];
-					[subscriber sendCompleted];
-				}
-			}];
-#pragma clang diagnostic pop
-			
-			return [RACDisposable disposableWithBlock:^{
-				// It's not clear if this will actually cancel the connection,
-				// but we can at least prevent _some_ unnecessary work --
-				// without writing all the code for a proper delegate, which
-				// doesn't really belong in RAC.
-				queue.suspended = YES;
-				[queue cancelAllOperations];
-			}];
-		}]
-		setNameWithFormat:@"+rac_sendAsynchronousRequest: %@", request];
+			 createSignal:^(id<RACSubscriber> subscriber) {
+				 NSURLSessionDataTask * task = [[NSURLSession sharedSession] dataTaskWithRequest:request completionHandler:^(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error) {
```

Comment:
This is still in `NSURLConnection+RACSupport.m`, so we'd have to move.

However, I'd say this is legacy code. I would stick to #2339 (minus changing `EMBEDDED_CONTENT_CONTAINS_SWIFT`), and if needed we can always add another extension for `NSURLSession` in Objective-C. We [already have one for Swift](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/master/ReactiveCocoa/Swift/FoundationExtensions.swift#L29-L50).